presto-raptor/src/test/java/com/facebook/presto/raptor/storage/TestOrcFileRewriter.java

```diff
@@ -159,6 +161,16 @@ public void testRewrite()
             assertTrue(arrayBlocksEqual(arrayType, arrayOfArrayType.getObject(column4, 4), arrayBlockOf(arrayType, arrayBlockOf(BIGINT, 9, 10))));
 
             assertEquals(reader.nextBatch(), -1);
+
+            OrcFileMetadata orcFileMetadata = orcFileMetadataCodec.fromJson(reader.getUserMetadata().get(OrcFileMetadata.KEY).getBytes());
+            assertEquals(orcFileMetadata, new OrcFileMetadata(ImmutableMap.<Long, TypeSignature>builder()
+                    .put(3L, BIGINT.getTypeSignature())
+                    .put(7L, VARCHAR.getTypeSignature())
```

Comment:
Same about `VARCHAR` constant. Better create some parametrized varchar types explicitly. Also you can add here some `DECIMAL` column.